src/sentry/tasks/auto_resolve_issues.py

```diff
@@ -60,8 +72,16 @@ def schedule_auto_resolution():
     silo_mode=SiloMode.REGION,
 )
 @log_error_if_queue_has_items
-def auto_resolve_project_issues(project_id, cutoff=None, chunk_size=1000, **kwargs):
+def auto_resolve_project_issues(
+    project_id,
+    enabled_issue_types: Optional[list[Type[grouptype.GroupType]]] = None,
```

Comment:
Actually, thinking this through a bit more, is there any need to pass this at all? Should we just fetch all the enabled group types in task? Saves changing the signature, and I'm not sure how well enums will serialize over json

Reply:
@wedamija Ah, so just get the enabled auto-resolve types locally in the auto_resolve_project_issues function itself?